Editor/src/gui/scenepanel.cpp

```diff
@@ -364,8 +364,82 @@ namespace Light
 				}
 				ImGui::EndCombo();
 			}
-
 			ImGui::Columns(1);
+
+			if(static_cast<int>(type)== 2)
+			{	
+				
+				component.m_inner = glm::clamp(component.m_inner, 0.0f, 90.0f);
```

Comment:
Just clamp the thing using GUI only (slider takes as input max and min values)
No need to clamp it using extra code as well

Reply:
the GUI clamps can be bypassed by typing an arbitrary value into the text box, so they are necessary.

Reply:
Okay
Clamp the value *if* the slider has been used, then

_Sent from my OnePlus 7 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
Done. NOTE: increasing the inner radius beyond the value of the outer, increases the outer as well till 90.00
Similarly for decreasing the outer radius, the inner also decreases.